tests/integration/targets/etcd3/runme.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
`runme.sh` is only needed for certain tests such as inventory or connection plugins. Especially for modules, it should not be used. This should be a role-based test (as the integration test for almost everything else).

Reply:
The chiken and egg problem: etcd3 module require etcd3 _python_ module to be installed first. It can't be done within a single _play_, that's why two playbooks are called sequentially:
* dependencies.yml
* test_etcd3.yml

Reply:
This is only true for plugins, not for modules.